sb_telemetry_util/telemetryService.js

```diff
@@ -1,13 +1,14 @@
 var Telemetry = require('./telemetryLibrary.js')
 var telemetrySyncManager = require('./telemetrySyncManager.js')
+const _ = require('lodash')
```

Comment:
Can you remove this lodash dependency? Not used anywhere

Reply:
ok